package.json

```diff
@@ -15,6 +15,7 @@
     "@mui/material": "^5.15.2",
     "@mui/styled-engine": "^5.15.2",
     "axios": "^1.6.3",
+    "formik": "^2.4.5",
```

Comment:
You are using react-hook-form, so you need to uninstall formik to reduce the bundle size.

Reply:
SORRY :'D